src/handlers/multiSig/multiSigExecuted/multiSigExecuted.ts

```diff
@@ -35,6 +37,29 @@ export const handleMultiSigMotionExecuted: EventHandler = async (
   const finalizedMultiSig = await getMultiSigFromDB(multiSigDatabaseId);
 
   if (finalizedMultiSig) {
+    const multiSigClient = await getMultiSigClient(colonyAddress);
+
+    if (!multiSigClient) {
+      return;
+    }
+
+    const motion = await multiSigClient.getMotion(motionId);
+
+    const actionData = motion.data[0];
+
+    if (!actionData) {
+      verbose(`No action data in multiSig motion: ${motionId}`);
+
+      return;
+    }
+
+    await linkPendingMetadata(
+      actionData,
+      colonyAddress,
+      finalizedMultiSig.id,
+      true,
+    );
+
     /* @TODO fix this up when we start porting over domain motions
```

Comment:
This comment can go now that you've implemented it

Reply:
Aye!